src/lib/spinel/radio_spinel.cpp

```diff
@@ -220,6 +220,10 @@ void RadioSpinel::InitializeCaps(bool &aSupportsRcpApiVersion, bool &aSupportsRc
 
 otError RadioSpinel::CheckRadioCapabilities(otRadioCaps aRequiredRadioCaps)
 {
+    static const char *const kAllRadioCapsStr[] = {"none",         "ack-timeout",     "energy-scan",   "tx-retries",
```

Comment:
Should remove "none" and start with "ack-timeout"?
Since index is mapped to bit position and bit 0 is used for ack-timeout?